samples/pubsub/native/sample.md

```diff
@@ -43,12 +43,12 @@ Click the `1` key repeatedly in the `Publisher` process console window and notic
 
 ## Message Flow
 
-The exact message flow may differ between [unicast transports](/transports/types.md#unicast-only-transports) and [broker transports](/transports/types.md#broker-transports). See the [publish-subscribe documentation](/nservicebus/messaging/publish-subscribe/) for further details.
+In multicast-enabled transports, the broker handles the subscription mechanism, routing published events to all subscribed endpoints. See the [publish-subscribe documentation](/nservicebus/messaging/publish-subscribe/#mechanics-native) for further details.
```

Comment:
```suggestion
In multicast-capable transports, the broker handles the subscription mechanism, routing published events to all subscribed endpoints. See the [publish-subscribe documentation](/nservicebus/messaging/publish-subscribe/#mechanics-native) for further details.
```

Reply:
I took "multicast-enabled" from here: https://docs.particular.net/transports/types#multicast-enabled-transports, is it worth to maintain a consistent language?